tests/console/systemd_testsuite.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's systemd tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Run testsuite included in systemd sources
+# Maintainer: Thomas Blume <tblume@suse.com>
+
+use base "consoletest";
+use warnings;
+use strict;
+use testapi;
+use utils;
+
+sub run() {
+    # install systemd testsuite
+    select_console 'root-console';
+    if (check_var('VERSION', '12-SP2')) {
```

Comment:
@tblume I think coolo's suggestion was to submit the test sweet to the standard QA:HEAD project in IBS where most such tests are kept. this would allow you to use the openQA QA_HEAD_REPO variable which is used in many places (eg. https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/6d147c60fd2f848a0a84d4c777ade2aac27fcf46/tests/qa_automation/qa_run.pm ) and includes the distro/version in question as part of that variable - meaning you can feed the test the correct repo at scheduling time and not have to put any 'if version' logic in your test.

We generally try and keep the tests as version/flavour generic as possible to encourage reuse - that's why both I and coolo don't like this check_var here